Added how-to on the topics feature selection and sample selection (#116)
Luthaf/rascaline: docs/src/how-to/feature-selection.rst

> Feature Selection

Reviewer: This should be renamed (this file, the example file and titles) to refer to `properties`, since that's the name equistore uses.

=================

> This examples shows how to compute certain SOAP power spectrum descriptor 
> properties for each atoms of each structures of a provided structure file, as well as 
> a methodology for selecting the most significant features with usage of farthest point sampling (FPS)
> algorithm within a single block. The path to the structure file is taken from the first command line
> argument.

Reviewer: ```suggestion
This examples shows how to only compute a subset of properties for each sample with 
a given rascaline representation. In particular, we will use the SOAP power spectrum representation, 
and select the most significant features within a single block using farthest point sampling (FPS). 
We will run the calculation for all atoms in a structure file, the path to which should be given as the first command line argument.
```

> This can be useful if we are interested in the contribution of individual features to the result, 
> or if we want to reduce the computational cost by using only part of the features for our model. 

Reviewer: ```suggestion
This is useful if we are interested in the contribution of individual features to the result, 
or if we want to reduce the computational cost by using only part of the features for our model. 
```
